examples/custom-plugin/plugin/bindings.ts

```diff
@@ -46,15 +52,11 @@ type __EventObj__<T> = {
 	once: (
 		cb: TAURI_API_EVENT.EventCallback<T>,
 	) => ReturnType<typeof TAURI_API_EVENT.once<T>>;
-	emit: T extends null
```

Comment:
Can you explain this change?

It also differs from the JS Doc alternative so they will need to match for this to be merged.

Reply:
to be honest i don't know when/how this changed. i will undo the change.